examples/java/README.md

```diff
@@ -0,0 +1,80 @@
+Download a protoc compiler (protoc-[version]-[Platform].zip) and and the protocol buffer for java zip or tar.gz (protobuf-java-[version]-zip/tar.gz) from the 
+[release page for protbuf](https://github.com/google/protobuf/releases). Make sure the version number is the same. Extract the files, go to the java directory and follow the instructions 
+in the [README](https://github.com/google/protobuf/blob/master/java/README.md). You have already done half of step two.
+
+#### Install Grpc
+Download [grpc-java](https://github.com/grpc/grpc-java) and in the grpc-java directory create a `gradle.properties` file with the text `skipCodegen=true`. 
+
+**In a command prompt:** go to the grpc-java directory, run `gradlew.bat build` and `gradlew.bat install`. This installs grpc in your local maven repository (usually located at `C:/Users/<user>/.m2`).
+
+## Changing the default logging
+Move the `javapluginlogger.properties` file to the `my-app/target` directory (it needs to be in the same directory as the `my-app-1.0-SNAPSHOT.jar` file).
+In this file you can edit the settings for the logging. There are different [levels](https://docs.oracle.com/javase/7/docs/api/java/util/logging/Level.html) of logging depending on how much 
+information you want (`OFF`, `SEVERE`, `WARNING`, `INFO`, `CONFIG`, `FINE`, `FINER`, `FINEST`, `ALL`).
+
+* `.level`: Sets the default level for all loggers, in this case they are set to info. 
+* `java.util.logging.ConsoleHandler.level`: Sets the level for the console handler, i.e. the level of logging sent to the console output (can't be more than the level of the loggers). 
+It is possible to add more handlers to different outputs (as an example you can add a file handler to send logging information to a file). 
+* `java.util.logging.ConsoleHandler.formatter`: Sets what format to use for the console output. 
+* `PluginServer.level`: Sets the level of the logger called PluginServer i.e. the logger used in the plugin.
+
+## Running the plugin with an insecure connection
+**In a command prompt:** go to the target directory and run `java -jar my-app-1.0-SNAPSHOT.jar`. This will start the java plugin listening on port 50071. If you want to specify which port 
+the plugin should listen to run `java -jar my-app-1.0-SNAPSHOT.jar --port <Port number>` instead.
+
+### Connecting to Qlik Sense Desktop
+Install Qlik Sense Desktop June release or later.
+To be able to connect to Sense you need to specify which port Sense should look for a plugin at. Go to `Documents/Qlik/Sense` and create or edit the `Settings.ini` file so that it includes 
+the line `SSEPlugin=javaPlugin,localhost:50071`. `javaPlugin` is the name of the plugin. This can be whatever you want but since the name of the plugin in the example app is set to `javaPlugin` the 
+example will not work if you set it to something else (unless you change the name of the plugin in every call to the plugin in the app). `50071` is the port. You should change this to `<Port number>` 
```

Comment:
Keep it short, like: "'javaPlugin' is an arbitrary name, that needs to match the name used in Sense, when referencing the plugin."